server/models/procedures.sql

```diff
@@ -1334,4 +1334,152 @@ BEGIN
   ON DUPLICATE KEY UPDATE `quantity` = `quantity` + movementQuantity;
 END $$
 
+-- This stored procedure, insert a lot record into the table
+CREATE PROCEDURE CreateLot (
+  IN uuid BINARY(16),
+  IN label VARCHAR(255),
+  IN initial_quantity INT(11),
+  IN unit_cost DECIMAL(19,4) UNSIGNED,
+  IN expiration_date DATE,
+  IN inventory_uuid BINARY(16),
+  IN origin_uuid BINARY(16),
+  IN delay INT(11)
+)
+BEGIN
+  INSERT INTO 
+    `lot` (`uuid`, `label`, `initial_quantity`, `unit_cost`, `expiration_date`, `inventory_uuid`, `origin_uuid`, `delay`)
+  VALUES
+    (uuid, label, initial_quantity, unit_cost, expiration_date, inventory_uuid, origin_uuid, delay);
+END $$
+
+
+-- This stored procedure, insert a record into the table stock movement
+CREATE PROCEDURE CreateStockMovement (
```

Comment:
Why do these need to be procedures?  It looks like all they do is `INSERT` to a regular table...

Both `CreateCash` and `CreateInvoice` only exist because they use `TEMPORARY` tables that need to be shared between stored procedures.  Since these procedures don't use TEMPORARY TABLEs, I don't think they need to exist .. they will just be one more thing to update when we try to do a database migration.